.eslintrc.js

```diff
@@ -149,5 +149,16 @@ module.exports = {
         'import/no-dynamic-require': 'off',
       },
     },
+    {
+      files: ['dev/**/*.js'],
+      rules: {
+        'node/no-unpublished-require': [
```

Comment:
We should remove this override and instead add `dev/` to `.npmignore`

Reply:
Done.